### DISCUSSION TITLE
Rotate only some column labels?

### DISCUSSION THREAD

**Discussion by rsmiller74** [rstudio/gt#1547] [Q&A]
I am attempting to rotate column labels but only want to do it for some columns.  Currently I am doing it in opt_css() but having problems with padding and rotating everything. Is there a way to do it as a tab_options?  I am currently using:

   opt_css(css = "#mygt .gt_col_heading {
                  text-align: center;
                  transform: rotate(-90deg)}"
          )

![2024-01-19_08-55-36](https://github.com/rstudio/gt/assets/7727644/0cef71f8-ea05-4599-868e-537b1f789a4f)

**Comment by mdgrant** [comment]
Try adding a style:

```
vertical-text {
   writing-mode: vertical-lr;
   transform: rotate(180deg);
}
```

A function for labels to rotate:

```
vert_lab <- function(label){
  paste0("<vertical-text>", label, "</vertical-text>")
}
```

```
cols_label(
  study_l = md("Study"),
  kq1_eras = md(vert_lab("ERAS")),
  gen_comp = md(vert_lab("CGA")),
  ...
```

<img width="782" alt="Screenshot 2024-01-24 at 5 01 04 PM" src="https://github.com/rstudio/gt/assets/10179515/4d7927cf-4d49-4822-96b8-88b34bf47d74">
